src/Namotion.Reflection/Context/ContextualType.cs

```diff
@@ -209,9 +209,9 @@ public Nullability Nullability
         /// </summary>
         /// <typeparam name="T">The attribute type.</typeparam>
         /// <returns>The attribute or null.</returns>
-        public T? GetContextAttribute<T>()
+        public T? GetContextAttribute<T>() where T : Attribute
         {
-            return ContextAttributes.OfType<T>().SingleOrDefault();
```

Comment:
SingleOrDefault<T>?

Reply:
now using `GetSingleOrDefault`, different name for easier to recognize as non-LINQ